Для помощи в отступе между flex-элементами

BentonFraizer opened the pull request:
Свёрстана секция "Команда", JS ещё не реализован. В процессе выполнения
стилизации.

Также в файл со шрифтами добавлны свойства font-display:swap;

BentonFraizer committed:
Для помощи в отступе между flex-элементами

Свёрстана секция "Команда", JS ещё не реализован. В процессе выполнения
стилизации.

Также в файл со шрифтами добавлны свойства font-display:swap;

BentonFraizer committed:
Реализована секция "Команда", добавлен 2-й слайд

Свёрстана и добавлен JS-функционал на секцию "Команда".
Также подкорректирован 2-й слайд в секции sliders-section.

BentonFraizer committed:
Реализован функционал слайдера секции "Отзывы"

Внесены мелки правки, связанные с адаптивом и корректным переключением
слайдов секции с отзывами.

BentonFraizer committed:
Для код ревью формы отправки заказа.

Промежуточный результат.

BentonFraizer committed:
Реализован функционал работы формы (JS).

В частности валидация и всплывающее модальное окно с сообщением ответа
сервера (успех/неудача).

BentonFraizer committed:
Подкорректировано значение margin у заголовка формы

b-ars07 commented:
1. "прыгает" слайдер при переключении - 
![image](https://user-images.githubusercontent.com/9081116/87697985-ec85a680-c7a3-11ea-8206-f925249362e1.png)
2. после успешной отправки форма должна отчищаться
3. утер не по макету (нет контейнера) - 
![image](https://user-images.githubusercontent.com/9081116/87698096-1048ec80-c7a4-11ea-99f9-8c90535cc6f5.png)
4. секция отзывов прыгает в мобилках при переключении - 
5. 
![image](https://user-images.githubusercontent.com/9081116/87698222-3d959a80-c7a4-11ea-9785-dc139a70a2e7.png)
![image](https://user-images.githubusercontent.com/9081116/87698259-48e8c600-c7a4-11ea-99e9-528bef1a8c3c.png)

BentonFraizer committed:
Устранены замечания наставника

А именно:
1. Устранено "скакание" слайдов;
2. После успешной отправки форма очищается;
3. Футер реализован согласно макета;
4. Секция отзывов на мобильных устройствах больше не "скачет";
5. Позвал недостающего человека в "Команду" :)
